database/data_seeding/query_generator.js

```diff
@@ -0,0 +1,8 @@
+module.exports.generateInsertQuery = (table, data) => {
+  const columns = Object.keys(data).join(', ');
+  const values = Object.values(data).join(', ');
+
+  const query = `INSERT INTO ${table} (${columns}) VALUES (${values});`;
+
+  return query;
```

Comment:
this is cool. i like the modularity of this.

Reply:
thanks!!